cirq-core/cirq/ops/measurement_gate.py

```diff
@@ -158,9 +158,12 @@ def _circuit_diagram_info_(
 
         # Mention the measurement key.
         if not args.known_qubits or self.key != _default_measurement_key(args.known_qubits):
-            symbols[0] += f"('{self.key}')"
+            if args.qubit_map is None or self.key not in args.qubit_map:
+                symbols[0] += f"('{self.key}')"
+        if args.qubit_map is not None:
```

Comment:
This is a weird condition. Revisit and make sure it's the right thing to do.

Reply:
We add this if there's a row, so seems like the condition should be if the key is in the qubit map.